packages/@aws-cdk/core/test/cloudformation-json.test.ts

```diff
@@ -1,12 +1,32 @@
-import { nodeunitShim, Test } from 'nodeunit-shim';
 import { App, CfnOutput, Fn, Lazy, Stack, Token } from '../lib';
 import { Intrinsic } from '../lib/private/intrinsic';
 import { evaluateCFN } from './evaluate-cfn';
 
-nodeunitShim({
-  'string tokens can be JSONified and JSONification can be reversed'(test: Test) {
-    const stack = new Stack();
+let app: App;
+let stack: Stack;
+beforeEach(() => {
+  app = new App();
+  stack = new Stack(app, 'Stack');
+});
+
+test('JSONification of literals looks like JSON.stringify', () => {
```

Comment:
I am wondering if we can pull out an existing test suite for JSON.stringify to ensure we cover the spec correctly...